### PR TITLE
Update us.m3u

### LINKED ISSUE #14470: Broken: BBC  America, IFC East, Sundance East
### Broken Link

https://bcovlive-a.akamaihd.net/c9bf201b06694453bb29282f97191f58/us-east-1/6240731308001/playlist.m3u8

### What happened to the stream?

Shows looped video

### Notes (optional)

IFC East
https://bcovlive-a.akamaihd.net/045bd3b615254da880c3db79b2e42fd0/us-east-1/6240731308001/playlist.m3u8

Sundance East
https://bcovlive-a.akamaihd.net/4d972ec6f41241f2b4286f8bdcc8dae9/us-east-1/6240731308001/playlist.m3u8

### Contributing Guide

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md)

### LINKED ISSUE #14482: Broken: Multiple Georgia Streams
### Broken Link

https://d2rwx6gwduugne.cloudfront.net/v1/master/77872db67918a151b697b5fbc23151e5765767dc/cmg_PROD_cmg-tv-10010_183ec1c7-4183-4661-803b-3ed282ffb625_LE/in/cmg-wsbtvnow-hls-v3/live.m3u8

### What happened to the stream?

Not loading

### Notes (optional)

All streams refuse to load with a 404 style non found error.

ABC 2 Atlanta GA (WSB-TV) (720p):
https://d2rwx6gwduugne.cloudfront.net/v1/master/77872db67918a151b697b5fbc23151e5765767dc/cmg_PROD_cmg-tv-10010_183ec1c7-4183-4661-803b-3ed282ffb625_LE/in/cmg-wsbtvnow-hls-v3/live.m3u8

CBS Atlanta GA (WGCL-DT1) (1080p):
https://trn10.tulix.tv/WGCL-CBS/index.m3u8

CW Network Atlanta-GA (WUPA-DT1) (1080p):
https://trn10.tulix.tv/WUPA-CW/index.m3u8

PBS Athens GA (WGTV) (1080p):
https://wgtvdt.lls.pbs.org/out/v1/1fe7fe6e36524881bd999a7003394ec7/index.m3u8

### Contributing Guide

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md)

### PR DESCRIPTION
**Pull request by freearhey**
Closes https://github.com/iptv-org/iptv/issues/14470, closes #14482

### CODE REVIEW

**Comment by BellezaEmporium** [comment]
Only https://github.com/iptv-org/iptv/blob/d526474567002f0e121290fbbb250b9c867b3ea9/streams/us_pbs.m3u#L124-L143 works.

**Comment by freearhey** [comment]
Updated.

File: streams/us_pbs.m3u [hunk 1]
```diff
@@ -141,147 +19,3 @@ https://livestream.pbskids.org/out/v1/19d1d62bf61b4aea9ec20f83b6450a4e/hast.m3u8
...
-#EXTINF:-1 tvg-id="WMVSDT1.us",PBS Milwaukee WI (WMVS) (1080p)
-https://wmvsdt.lls.pbs.org/out/v1/654e3fa9db6d465ea578cf39818fcee6/index.m3u8
-#EXTINF:-1 tvg-id="KTCADT1.us",PBS Minneapolis MN (KTCA) (1080p)
-https://ktcadt.lls.pbs.org/out/v1/21103812ea504393b7f0d521a8b37ab7/index.m3u8
-#EXTINF:-1 tvg-id="WQPTDT1.us",PBS Moline IL (WQPT) (1080p)
-https://wqptdt.lls.pbs.org/out/v1/6ab72cd3813b469cb5f79a577d49c0b7/index.m3u8
-#EXTINF:-1 tvg-id="WCMUDT1.us",PBS Mount Pleasant MI (WCMU) (1080p)
-https://wcmudt.lls.pbs.org/out/v1/45ff0524571c41398c5f39ebab6262ef/index.m3u8
-#EXTINF:-1 tvg-id="WMPTDT1.us",PBS MPT Baltimore MD (WMPB) (1080p)
-https://2-fss-2.streamhoster.com/pl_138/amlst:201814-1291584/playlist.m3u8
```
Comment (BellezaEmporium) [inline streams/us_pbs.m3u:185]: This one works too. My bad.

File: streams/us_pbs.m3u [hunk 2]
```diff
@@ -141,147 +19,3 @@ https://livestream.pbskids.org/out/v1/19d1d62bf61b4aea9ec20f83b6450a4e/hast.m3u8
...
-#EXTINF:-1 tvg-id="WIPBDT1.us",PBS Muncie IN (WIPB) (1080p)
-https://wipbdt.lls.pbs.org/out/v1/73cd2d86797b4fc6ac5b660cd5a6f9c4/index.m3u8
-#EXTINF:-1 tvg-id="WNPTDT1.us",PBS Nashville TN (WNPT) (1080p)
-https://wnptdt.lls.pbs.org/out/v1/f6bca6d722674c5cad621f54b17c217b/index.m3u8
-#EXTINF:-1 tvg-id="PBSEast.us",PBS National East (1080p) [Geo-blocked]
-https://pbs.lls.cdn.pbs.org/est/index.m3u8
```
Comment (BellezaEmporium) [inline streams/us_pbs.m3u:191]: This one works too. My bad.

File: streams/us_pbs.m3u [hunk 3]
```diff
@@ -141,147 +19,3 @@ https://livestream.pbskids.org/out/v1/19d1d62bf61b4aea9ec20f83b6450a4e/hast.m3u8
...
-#EXTINF:-1 tvg-id="PBSWest.us",PBS National West (1080p) [Geo-blocked]
-https://pbs.lls.cdn.pbs.org/pst/index.m3u8
```
Comment (BellezaEmporium) [inline streams/us_pbs.m3u:193]: This one works too. My bad.

File: streams/us_pbs.m3u [hunk 4]
```diff
@@ -141,147 +19,3 @@ https://livestream.pbskids.org/out/v1/19d1d62bf61b4aea9ec20f83b6450a4e/hast.m3u8
...
-#EXTINF:-1 tvg-id="WHUTDT1.us",PBS Washington DC (WHUT) (1080p)
-https://whutdt.lls.pbs.org/out/v1/dd1102f24d7948e58517ba6a6573c928/index.m3u8
-#EXTINF:-1 tvg-id="WPBSDT1.us",PBS Watertown NY (WPBS) (1080p)
-https://wpbsdt.lls.pbs.org/out/v1/e3680a91029c4df9b7797ce13d828207/index.m3u8
-#EXTINF:-1 tvg-id="WXELDT1.us",PBS West Palm Beach FL (WXEL) (1080p)
-https://wxeldt.lls.pbs.org/out/v1/d4f2bc8357164a2e93d35abd2caecc4b/index.m3u8
-#EXTINF:-1 tvg-id="KPTSDT1.us",PBS Wichita KS (KPTS) (1080p)
-https://kptsdt.lls.pbs.org/out/v1/1ca3404d6bc3404c9daa86c8f1ae19d0/index.m3u8
-#EXTINF:-1 tvg-id="WMPTDT2.us",WMPT-DT2 (PBS Create)
-https://2-fss-1.streamhoster.com/pl_122/amlst:201814-2122456/playlist.m3u8
```
Comment (BellezaEmporium) [inline streams/us_pbs.m3u:289]: This one works too. My bad.

**Review by BellezaEmporium — COMMENTED** [review]
Found out HLS links works... if you match the IP to the PBS station. How odd. I remember they used to work everywhere as long as you had an US IP. Now it's even more restrictive.

**Comment by freearhey** [comment]
Updated.

**Review by BellezaEmporium — DISMISSED** [review]
Reserved approval.

While we're updating US channels : I have a few here : 

https://wway.akamaized.net/hls/live/2029583/wway_ingest/master.m3u8 ABC WWAY 3 (WWAY-DT1) Wilmington, North Carolina
https://vbfast-c.viebit.com/810fe13b-a9ac-46b3-a523-5a7e90ddfd25/playlist.m3u8 Access La Porte Country Channel 97 Michigan City, Indiana
https://livestream.telvue.com/nashuanh1/f7b44cfafd5c52223d5498196c8a2e7b.sdp/playlist.m3u8 Access Nashua Nashua, New Hampshire
https://livestream.telvue.com/nashuanh2/f7b44cfafd5c52223d5498196c8a2e7b.sdp/playlist.m3u8 Nashua ETV Channel 22 Nashua, New Hampshire
https://livestream.telvue.com/nashuanh3/f7b44cfafd5c52223d5498196c8a2e7b.sdp/playlist.m3u8 Nashua GTV Channel 16 Nashua, New Hampshire
https://livestream.telvue.com/accvision1/f7b44cfafd5c52223d5498196c8a2e7b.sdp/playlist.m3u8 AccessVision Channel 16 Battle Creek, Michigan
https://livestream.telvue.com/accvision2/f7b44cfafd5c52223d5498196c8a2e7b.sdp/playlist.m3u8 AccessVision Channel 17 Battle Creek, Michigan

**Comment by BellezaEmporium** [comment]
Asking @PopeyeTheSai10r here to verify the theory I have over the PBS HLS links.

**Comment by freearhey** [comment]
Added.

**Comment by PopeyeTheSai10r** [comment]
> Asking @PopeyeTheSai10r here to verify the theory I have over the PBS HLS links.

Some of the streams/us_pbs.m3u links are still working but they seen to be region-locked. 

The following links worked for me while using a NY IP. They won't work when using US IP from other states.
```
https://wnetdt.lls.pbs.org/out/v1/0456457548354b88b32fc437e4e7ee01/index.m3u8
https://wnjtdt.lls.pbs.org/out/v1/e62efd8d4f92403996425fc389df0ffd/index.m3u8
```

**Comment by freearhey** [comment]
> The following links worked for me while using a NY IP. They won't work when using US IP from other states.
>
> https://wnetdt.lls.pbs.org/out/v1/0456457548354b88b32fc437e4e7ee01/index.m3u8
> https://wnjtdt.lls.pbs.org/out/v1/e62efd8d4f92403996425fc389df0ffd/index.m3u8

Brought those links back.

**Comment by BellezaEmporium** [comment]
I'll check with my VFN over the available regions and come back with an answer.

File: streams/us.m3u [hunk 5]
```diff
@@ -1139,4 +1133,18 @@ http://190.11.225.124:5000/live/playboy_hd/playlist.m3u8
 #EXTINF:-1 tvg-id="PenthouseGold.us",Penthouse Gold (1080p)
 http://51.159.102.100/PenthouseGold/index.m3u8
 #EXTINF:-1 tvg-id="BrazzersTVEurope.us",Brazzers TV Europe (720p)
-https://cdn-main.lolokoko.tv/BRAZZERS.stream/chunks.m3u8
+https://cdn-global.lolokoko.tv/BRAZZERS.stream/chunks.m3u8
```
Comment (BellezaEmporium) [inline streams/us.m3u:1136]: I believe that wasn't intentional.

File: streams/us_pbs.m3u [hunk 6]
```diff
@@ -1,128 +1,4 @@
 #EXTM3U
-#EXTINF:-1 tvg-id="KETSDT1.us",Arkansas PBS [Geo-blocked]
-https://ketsdt.lls.pbs.org/out/v1/03c094dbd7874a4a8c3fe9fb10081bdb/index.m3u8
```
Comment (BellezaEmporium) [inline streams/us_pbs.m3u:3]: Works (VPN set to Buffalo & New York)

File: streams/us_pbs.m3u [hunk 7]
```diff
@@ -1,128 +1,4 @@
...
-#EXTINF:-1 tvg-id="WFWADT1.us",PBS Fort Wayne IN (WFWA) (1080p)
-https://wfwadt.lls.pbs.org/out/v1/8b2a780393274c2ba9b71b9168df2208/index.m3u8
-#EXTINF:-1 tvg-id="WYINDT1.us",PBS Gary IN (WYIN) (1080p)
-https://wyindt.lls.pbs.org/out/v1/27414b46e52d43758be8227e4a91c863/index.m3u8
-#EXTINF:-1 tvg-id="WITFDT1.us",PBS Harrisburg PA (WITF) (1080p)
-https://witfdt.lls.pbs.org/out/v1/15cd55cd6f7442c4a193a47bbfae608a/index.m3u8
-#EXTINF:-1 tvg-id="WEDHDT1.us",PBS Hartford CT (WEDH) (1080p)
-https://wedhdt.lls.pbs.org/out/v1/04182c8d6be24c1a98de212f3c55a442/index.m3u8
-#EXTINF:-1 tvg-id="KHETDT1.us",PBS Honolulu HI (KHET) (1080p) [Not 24/7]
-https://khetdt.lls.pbs.org/out/v1/7ec7903413294b72bb64f83963d8ea9b/index.m3u8
```
Comment (BellezaEmporium) [inline streams/us_pbs.m3u:115]: Works (VPN set to Buffalo & New York)

File: streams/us_pbs.m3u [hunk 8]
```diff
@@ -143,147 +19,15 @@ https://livestream.pbskids.org/out/v1/19d1d62bf61b4aea9ec20f83b6450a4e/hast.m3u8
...
-#EXTINF:-1 tvg-id="KLVXDT1.us",PBS Las Vegas NV (KLVX) (1080p)
-https://klvxdt.lls.pbs.org/out/v1/c2d457573e4d4783b558c44e20547e21/index.m3u8
-#EXTINF:-1 tvg-id="WCBBDT1.us",PBS Lewiston ME (WCBB) (1080p)
-https://wcbbdt.lls.pbs.org/out/v1/d1ca1cb603fd4da09694f0f1f6ba8db0/index.m3u8
-#EXTINF:-1 tvg-id="WKLEDT1.us",PBS Lexington KY (WKLE) (1080p)
-https://wkledt.lls.pbs.org/out/v1/26354efcebc1400e8861988cd6a321ca/index.m3u8
-#EXTINF:-1 tvg-id="WLJTDT1.us",PBS Lexington TN (WLJT) (1080p)
-https://wljtdt.lls.pbs.org/out/v1/1ddf810d67f64eeeab40ad9da8885945/index.m3u8
-#EXTINF:-1 tvg-id="KUONDT1.us",PBS Lincoln NE (KUON) (1080p) [Geo-blocked]
-https://kuondt.lls.pbs.org/out/v1/91d8b5ffc5c1453c8a621508a07749a6/index.m3u8
```
Comment (BellezaEmporium) [inline streams/us_pbs.m3u:161]: Works (VPN set to Buffalo & New York)

File: streams/us.m3u (hunk 5 above)
Comment (freearhey) [reply streams/us.m3u]: Updated.

**Comment by freearhey** [comment]
Updated.

**Review by BellezaEmporium — APPROVED** [review]

**Review by Alstruit — APPROVED** [review]

**Pull request merged**